agreement/agreementtest/simulate.go

```diff
@@ -19,6 +19,7 @@ package agreementtest
 
 import (
 	"fmt"
+	"github.com/algorand/go-algorand/protocol"
```

Comment:
Could you move this import to where the other `github.com/algorand/go-algorand` imports are?

Reply:
Is there a script that can check for these? I feel like this is happening because my editor auto imports things

Reply:
Hmm, that's a good question. If `make sanity` doesn't do anything, then probably not currently...